src/dialogs/examples/basic/AdaptiveCardDialog.ts

```diff
@@ -0,0 +1,451 @@
+import * as builder from "botbuilder";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+
+export class AdaptiveCardDialog extends TriggerActionDialog {
+
+    private static async step1(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void>
```

Comment:
Let's rename "step1" to something like "sendAdaptiveCard"

Reply:
Done.